Makefile

```diff
@@ -134,6 +134,8 @@ clean-sql-files:
 
 clean-extra:
 	@rm -f src/*~
+	@rm -f src/*.d
```

Comment:
I frequently pull before running clean so yeah I guess I run into it more often.